spec/models/odk/response_parser_spec.rb

```diff
@@ -160,27 +172,54 @@
         end
 
         context "with invalid data types" do
-          let(:xml_values) { ["9.6", "The quick brown fox", "foo", "bar"] }
+          let(:xml_values) { ["9.6", "The quick brown fox", "foo", ""] }
```

Comment:
A consequence of verifying that the option node is in the current mission is that invalid option node IDs (like `bar`) will now fail. Is this acceptable? Seems reasonable unless we want to handle something like old forms being submitted with since-deleted values.